BrightFuturesTests/ResultTests.swift

```diff
@@ -21,8 +21,18 @@
 // SOFTWARE.
 
 import XCTest
+import Result
 import BrightFutures
 
+extension Result {
+    var isSuccess: Bool {
```

Comment:
The only place these were used extensively was in these tests, that's why I created a local extension on `Result` for it. The only other place is on `Future.isSuccess` and `Future.isFailure`, where they're implemented in the same way now.